lib/pages/kyouen/kyouen_page.dart

```diff
@@ -19,9 +19,9 @@ class KyouenPage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final KyouenPageArguments args =
-        ModalRoute.of(context).settings.arguments as KyouenPageArguments;
-    final String roomId = args.roomId;
+    final args =
+        ModalRoute.of(context)?.settings.arguments as KyouenPageArguments;
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/kyouen_vs_flutter/blob/c7f5fbcba3dc8b07bf8b90fa42ebfb4e9f206c26/lib/pages/kyouen/kyouen_page.dart#L23"></span>Don't cast a nullable value to a non nullable type</td>
    </tr>
  </tbody>
</table>